src/services.ts

```diff
@@ -43,7 +44,13 @@ function autoSaveModels (services: CgMonacoServices) {
         console.error('[LSP]', `Unable to save the document ${e.textDocument.uri.toString()}`, err)
       })
     }, 500))
-  })
+  }))
+  disposableCollection.push(Disposable.create(() => {
+    for (const timeout of Array.from(timeoutMap.values())) {
```

Comment:
Why do you need this Array.from?

Reply:
Or else there is an eslint error (`.values()` return an iterator)